obspy/signal/spectral_estimation.py

```diff
@@ -617,6 +607,26 @@ def __process(self, tr):
         # working with the periods not frequencies later so reverse spectrum
         spec = spec[::-1]
 
+        # Here we remove the response using the same conventions
+        # since the power is squared we want to square the sensitivity
+        # we can also convert to acceleration if we have non-rotational data
+        if self.is_rotational_data:
+            # in case of rotational data just remove sensitivity
+            spec /= paz['sensitivity']**2
```

Comment:
It's not really a requirement for PEP8; not sure we do that consistently either.